### PR TITLE
Added Quicksort in C++ fixes issue #12 .

### PR DESCRIPTION
**Pull request by Roshan13046**
Issue no #12  fixed.
Quicksort algorithm in c++ added.
Please check.
Thank you.

### CODE REVIEW

File: Sorting/quick_sort/quick_sort.cpp.cpp [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	}
+}
+
+
+//main function
+int main(){
+	
+	int arr[]= {2,4,3,6,9,8, -1};	
+	int n = sizeof(arr)/sizeof(arr[0]);
+
```
Comment (nikochiko) [inline Sorting/quick_sort/quick_sort.cpp.cpp]: Can we take this input from STDIN.
First line: N - number of inputs
Second line: N space-separated integers

for example, 
```
5
2 5 23 14 4
```

**Review by nikochiko — CHANGES_REQUESTED** [review]
Nice work! Few small changes and this will be ready to go.
Apart from the mentioned comment, also change the filename from `quick_sort.cpp.cpp` to `quick_sort.cpp`.

**Comment by Roshan13046** [comment]
> Nice work! Few small changes and this will be ready to go.
> Apart from the mentioned comment, also change the filename from `quick_sort.cpp.cpp` to `quick_sort.cpp`.

Done the suggested changes.
Please check.
Thanks.

**Review by nikochiko — APPROVED** [review]
Great job! :+1:

**Comment by Roshan13046** [comment]
> Great job! 👍

Thanks.

**Comment by nikochiko** [comment]
merged restyled version - https://github.com/nikochiko/cool-algos/pull/45. thanks for contributing

**Pull request closed without merging**